scss/layout/_scaffolding.scss

```diff
@@ -0,0 +1,122 @@
+@import '../abstracts/variables';
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+[class*='container'] {
+  margin: 0 auto;
+  padding-right: $column-gutter;
+  padding-left: $column-gutter;
+}
+.container-small {
+  max-width: $device-sm;
+}
+.container {
+  position: relative;
+  max-width: 66rem;
+}
+.container-large {
+  position: relative;
+  max-width: $device-lg;
+}
+
+.row {
```

Comment:
Again would want to check this class name for usage in webb3 to make sure we don't screw up any existing alignments.